astropy/table/jsviewer.py

```diff
@@ -168,23 +158,23 @@ def html_js(self, table_id='table0'):
                                      tid=table_id).strip()
 
 
-def write_table_jsviewer(table, filename, table_id=None,
-                         css="table,th,td,tr,tbody {border: 1px solid black; border-collapse: collapse;}",
-                         max_lines=5000,
-                         jskwargs={}):
+def write_table_jsviewer(table, filename, table_id=None, max_lines=5000,
```

Comment:
The `max_lines` arg isn't being used anywhere and I was able to put a 10000 line table into the browser (which was really slow).